src/accessors/IIterator.ts

```diff
@@ -1,6 +1,6 @@
-import { IIteratorResult } from './IIteratorResult';
-
 export interface IIterator<T> {
-    next(skip: number): IIteratorResult<T>;
+    next(packageSize?: number): Array<T>;
```

Comment:
@graywolf336 

- I actually prefer the pagesize on initialization of the constructor. Why do you have it as param in `next()`?
- Returning another wrapper from the `next()` seems less consumable from my pos